R/metrics_cc.R

```diff
@@ -0,0 +1,47 @@
+#' CC license variants metrics overview
+#'
+#' Presents normalised CC licence variants like BY (absolute and relative)
+#'
+#' @inheritParams metrics_overview
+#'
+#' @importFrom dplyr count mutate rename
```

Comment:
these are generally not necessary when using `dplyr::mutate()`.
I think `@importFrom` statements in the roxygen blocks for *individual* functions is also a bit confusing/misleading, because the resulting `NAMESPACE` directives actually affect the entire package.

So, I think when it comes to using functions from other packages, there's basically two scenarios:

1. If a *lot* of our functions use, say, `pkg::foo()`, then we should declare `@importFrom pkg foo` *once* in `metacheck.R` (conventionally used for that purpose)
2. If only *one*/few of our functions use, say `pkg::bar()`, then should simply always use `pkg::bar()`, and never `@importFrom`.

For dplyr etc., I think it's 1. We're using that all over the place, and I have `@import` it in #69, which I'll merge in here.
 `@import`ing from other packages, there

Reply:
Some style guides are discouraging use of the @import tag. https://google.github.io/styleguide/Rguide.html#qualifying-namespaces , but can follow the above scenarios.